tools/heir-opt.cpp

```diff
@@ -503,6 +504,7 @@ int main(int argc, char **argv) {
   registerElementwiseToAffinePasses();
   registerSecretizePasses();
   registerFullLoopUnrollPasses();
+  registerIfTransformPasses();  // Register IfTransfrom pass
```

Comment:
I think this comment is a bit redundant ;)
```suggestion
  registerIfTransformPasses();
```